src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions.mdx

```diff
@@ -0,0 +1,342 @@
+
+</Collapser>
+</CollapserGroup>
+</Step>
+
+<Step>
+## Fine-tune advanced signal settings [#advanced-signal-settings]
+
+New Relic constantly observes the data that streams from your applications and into our system. But not all applications send signals at the same frequency or cadence. Some events could send signals to our system once a minute while others could only report data to New Relic once every day. An alert condition is a **specific** container designed for a **specific** use case. When creating an alert condition this section is the most customizable for the data that you're evaluating. 
+
+We're going to customize these advanced signal settings for our condition that is montioring `pageviews` in the `WebPortal` application. 
+
+<img
+    title="fine tune alert settings"
+    alt="A screenshot depicting advanced signal settings"
+    src={alertsFineTuneAlertSignals}
+/>
+
+<CollapserGroup>
+  <Collapser
+    id="window-duration"
+    title="Window duration"
+  >
+
+  Setting the window duration for your alert condition tells New Relic how to group your data. If you're creating an alert condition for a data set that sends a signal to New Relic once every hour, you'd want to set the window duration to something closer to sixty minutes because it'll help spot patterns and unusual behavior. But, if you're creating an alert condition to monitor `pageviews` on your `WebPortal` application and New Relic collects a signal for that data every minute, we'd recommend setting the window duration to one minute. 
+  
+  When you're getting started we recommend sticking with our default settings, but the more you get familiar with creating an alert condition we encourage you to customize these fields based on your own experience. 
+  </Collapser>
+
+  <Collapser
+    id="sliding-window"
+    title="Use sliding window aggregation"
+  >
+
+  Throughout the day, data streams from your application into New Relic. Instead of evaluating that data immediately for incidents, alert conditions collect the data over a period of time known as the **aggregation window**. An additional delay allows for slower data points to arrive before the window is aggregated.
+
+  Sliding windows are helpful when you need to smooth out "spiky" charts. One common use case is to use sliding windows to smooth line graphs that have a lot of variation over short periods of time in cases where the rolling aggregate is more important than aggregates from narrow windows of time.
+
+  We recommend using our sliding window aggregation if you're not expecting to have a steady and consistent stream of data but are expecting some dips and spikes in data. 
+
+  You can learn more about sliding window aggregation in [this NRQL tutorial](/docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/create-smoother-charts-with-sliding-windows/) or by watching this video.
+
+  <Video
```

Comment:
honest i'd cut the videos and consider using bulleted lists that are skimmable for the collapsers. right now it's a lot of text and can be overwhelming or give the impression that it's very complex. if we can keep the structure simple, then customers might be more emboldened to try it out